_projects/nested-languages.md

```diff
@@ -59,11 +110,12 @@ advanced path would be to come up with a more abstract way to define how arbitra
 ##### CODING PHASE 2
 
 * At least three common combinations of languages are supported.
-* OR if applicable an abstract method of combining languages has been developed
-  and at least one bear exists which uses this method.
+* OR if applicable an abstract method of combining languages has been
+developed and at least one bear exists which uses this method.
```

Comment:
this decreases the style of the document.  they were intentionally indented.

and please dont change things unless it is part of your task.  'fix line lengths' is not your task.

Reply:
Yes thats by mistake that was only because I was adjusting the lines I'll fix it.